src/autocomplete/DuckDuckGoProvider.js

```diff
@@ -1,4 +1,21 @@
+/*
```

Comment:
So swap:
```
Copyright 2017 Vector Creations Ltd
```
with:
```
Copyright 2016 Aviral Dasgupta
```
sound okay? Or should we also put a 2017 Vector Creations along with it?